R/zzz-deprecate.R

```diff
@@ -586,3 +586,29 @@ deprecated("igraph.shape.noclip", shape_noclip)
 deprecated("igraph.shape.noplot", shape_noplot)
 #' @export create.communities
 deprecated("create.communities", make_clusters)
+
+
+#' @export
+graph.count.isomorphisms.vf2 <- graph.count.isomorphisms.vf2_impl
```

Comment:
@krlmlr could you do these calls in another script? I'm planning to delete zzz-deprecate. Otherwise I'll adapt my script, as you prefer.

Reply:
Thanks, done.